README.rst

```diff
@@ -42,14 +42,33 @@ A client at www.foo.com can use lizard-apps deployed at apps.lizard.net as follo
         <script src="http://apps.lizard.net/lizard_apps/screens/foo/" async></script>
       </head>
       <body>
-        <div id="lizard-apps-container" style="margin: 50px auto;"></div>
+      <div id="lizard-apps-button"></div>
```

Comment:
Moet je de lizard-apps-button in dit voorbeeld geen height en width geven? Zonder dit ziet het er belachelijk uit: svg all over de screen. Mensen kunnen die height en width dan naar wens aanpassen.